### DISCUSSION TITLE
I can't change the text color using CSS

### DISCUSSION THREAD

**Discussion by myxu13** [ExperienceLovelace/ha-floorplan#135] [Q&A]
I can't change the text color using CSS
`- service: floorplan.class_set
  service_data:
    element: Navigate2_Text
    class: Navigate_Text`
`.Navigate_Text {fill:#ff0000;}`

**Comment by exetico** [comment]
Please provide a little more information... Also while doing so, share your SVG. It can be a question about how you've layered it.

Also, format the text, so it's a bit easier to read 😊!

Also, check the other questions about fill. If I remember correct, there should be at least one of them 👍.

**Reply by OzGav** [reply]
Hi myxu13, Have you got this working? If so please mark it as answered. If not can you provide some more info as exetico suggested?
